### ISSUE TITLE
Add some dummy data to import for dev install

### ISSUE THREAD

**Issue by asmaloney** [bookwyrm-social/bookwyrm#2583]
**Is your feature request related to a problem? Please describe.**

I followed the dev installation from [here](https://docs.joinbookwyrm.com/install-dev.html) which gives me a clean install with just an admin user. 👍 

**Describe the solution you'd like**

It would be useful to have some default dummy data available - maybe a few users, each with some books, a list or two, etc.. Basically a couple of each "thing".

It could be added using another command: `./bw-dev install-dev-data` or something.

This would get people up-and-running with everything they need to jump into the code pretty quickly.

(Aside: The email setup should probably be addressed in the dev setup docs.)

**Describe alternatives you've considered**

I could not find a way to create users from the admin dashboard, so I had to log out and try to create a user. Then I realized I would need to set up mailgun.

I could probably figure that out, create users using multiple real email addresses, then presumably add a bunch of data by hand... 

... but it would be much more convenient to have some dummy data to import.

**Comment by mouse-reeve** [comment]
I agree! As a workaround in the meantime, you can go to the admin site registration admin panel and uncheck "Require users to confirm email address" to create new users without having to deal with email

**Comment by asmaloney** [comment]
Oh, cool. I hadn't found that. Thank you. 👍
